src/cli-client/internals/cli/cli_main.go

```diff
@@ -0,0 +1,66 @@
+package cli
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/canonical/go-flags"
+	"github.com/canonical/oci-factory/src/cli-client/internals/logger"
+)
+
+type CmdRelease struct {
+	Revision int `long:"revision" description:"The revision number of the image release"`
+}
+
+const longDescription = `The OCI Factory CLI client is a tool that builds, tests, and releases
+the OCI images owned by Canonical using the Github workflow in 
+the OCI Factory repository.
+`
+
+var opts struct {
+	// TODO: Leave `release` here for now. Should be moved into a `cmd_release.go` in phase 2.
+	// CmdRelease       CmdRelease `command:"release" description:"Release (re-tag) the image into the container registries"`
+	SkipConfirmation bool `short:"y" description:"Skip the confirmation to upload/release an image"`
+}
+
+var parser = flags.NewParser(&opts, flags.Default)
+
+func CliMain() {
+	if len(os.Args) == 1 || os.Args[1] == "help" {
+		// TODO: Add subcommand `help` if more complex help messages are needed
+		fmt.Print(longDescription + "\n")
+		parser.WriteHelp(os.Stdout)
+		os.Exit(0)
+	}
```

Comment:
How about the following:
```go
	addHelp(parser)
	if len(os.Args) == 1 {
		parser.WriteHelp(os.Stdout)
		os.Exit(0)
	}
```

And in this file we can add the `addHelp` method:
```go
func addHelp(parser *flags.Parser) {
	parser.ShortDescription = "CLI client to build, test and release OCI images"
	parser.LongDescription = strings.TrimSpace(longDescription)
}
```
We can rename `addHelp` to something more meaningful, like `configParser`.

Reply:
Agree! I think we can keep it as `addHelp` for now as it's not dealing with something more than adding the help messages.